keyboards/skyloong/dt40/dt40.c

```diff
@@ -0,0 +1,60 @@
+/* Copyright 2021 JZ-Skyloong (@JZ-Skyloong)
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+
+
+#include "quantum.h"
+
+#ifdef RGB_MATRIX_ENABLE
+#define RGB_MATRIX_LED_COUNT 44
```

Comment:
This doesn't do anything here.
```suggestion
```

And the led config should be moved to info.json (`qmk info -kb skyloong/dt40 -f json` should help with that)